spec/browsers/edge_user_agent_spec.rb

```diff
@@ -53,7 +57,31 @@
     expect(@useragent.version).to eq("74.1.96.24")
   end
 
+  it "should return 'Windows' as its platform" do
+    expect(@useragent.platform).to eq("Windows")
+  end
+
   it "should return 'Windows 10' as its os" do
     expect(@useragent.os).to eq("Windows 10")
   end
 end
+
+describe "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.96 Safari/537.36 Edg/88.0.705.56" do
```

Comment:
Can you add a spec for the Android and ChromeOS user agents as well?

Reply:
I've done Android and iOS - I didn't think Edge was available for ChromeOS, would have to do a little more hunting for a suitable user-agent string.

Reply:
As far as I can tell, installing Edge on ChromeOS would mean installing the Android app, so I think the Android cases should suffice!